File: app/src/main/java/com/orange/androidopenggltutorials/MainActivity.java
package com.orange.androidopenggltutorials;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.orange.androidopenggltutorials.render.camera.Camera2Wrapper;
import com.orange.androidopenggltutorials.render.camera.CameraPreviewActivity;
import com.orange.androidopenggltutorials.render.shape.ShapeActivity;

/**
 * created by czh on 2022.4.24
 * @author 19b300
 */
public class MainActivity extends AppCompatActivity {

   @Override
   protected void onCreate(@Nullable Bundle savedInstanceState) {
      super.onCreate(savedInstanceState);
      setContentView(R.layout.activity_main);
      initListener();
   }


   private void initListener(){
      findViewById(R.id.btDraw).setOnClickListener(new View.OnClickListener() {
         @Override
         public void onClick(View v) {
            startActivity(new Intent(MainActivity.this, ShapeActivity.class));
         }
      });

      findViewById(R.id.btCamera).setOnClickListener(new View.OnClickListener() {
         @Override
         public void onClick(View v) {
            startActivity(new Intent(MainActivity.this, CameraPreviewActivity.class));
         }
      });
   }
}

File: app/src/main/java/com/orange/androidopenggltutorials/MainActivity.kt
package com.orange.androidopenggltutorials

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import com.orange.androidopenggltutorials.render.shape.NativeLib
import com.orange.androidopenggltutorials.render.shape.ShapeActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        initListener()

        Log.d("czh","from native:"+ NativeLib.getTestString())
    }


    private fun initListener(){
        btDraw.setOnClickListener {
            startActivity(Intent(this, ShapeActivity::class.java))
        }


    }
}
